packages/app-project/src/interbit/my-projects/index.js

```diff
@@ -1,8 +1,8 @@
 // © 2018 BTL GROUP LTD -  This package is licensed under the MIT license https://opensource.org/licenses/MIT
 const Immutable = require('seamless-immutable')
 const {
-  selectors: interbufferSelectors,
-  covenant: { reducer: interbufferReducer }
+  manifestSelectors,
```

Comment:
perhaps rootSelectors? The manifestSelectors exist to select information from the manifest.json file elsewhere as well.